docker/Dockerfile.devcontainers

```diff
@@ -0,0 +1 @@
+FROM ghcr.io/dependabot/dependabot-updater-devcontainers:v2.0.20241017235241@sha256:5b854616a508dc1f3e759e0b3ba7abb7ee920797250bbdb2bbb8a1b47a4d43fc
```

Comment:
I think this has to be updated to `v2.0.20241001115348` too?

Reply:
Thanks and updated